docs/features/backup_and_restore.md

```diff
@@ -0,0 +1,116 @@
+---
+id: backup_and_restore
+title: Cumulus Backup and Restore
+hide_title: true
+---
+
+# Deployment Backup and Restore
+
+Most of your Cumulus deployment can be recovered by redeploying via Terraform. The Cumulus metadata including providers, collections, granules, rules, and executions are stored in [DynamoDB](./data_in_dynamodb) and can be setup to be backed up and restored. If a deployment is lost, logs and Step Function executions in the AWS console will be irrecoverable.
+
+## Backup and Restore with AWS
+
+You can enable [point-in-time recovery (PITR)](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/PointInTimeRecovery.html) as well as create an [on-demand backup](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/BackupRestore.html) for your Amazon DynamoDB tables.
+
+PITR provides continuous backups of your DynamoDB table data. PITR can be enabled through your Terraform deployment, the AWS console, or the AWS API. When enabled, DynamoDB maintains continuous backups of your table up to the last 35 days. You can recover a copy of that table to a previous state at any point in time from the moment you enable PITR, up to a maximum of the 35 preceding days. PITR provides continuous backups until you explicitly disable it.
+
+On-demand backups allow you to create backups of DynamoDB table data and its settings. You can initiate an on-demand backup at any time with a single click from the AWS Management Console or a single API call. You can restore the backups to a new DynamoDB table in the same AWS Region at any time.
+
+PITR gives your DynamoDB tables continuous protection from accidental writes and deletes. With PITR, you do not have to worry about creating, maintaining, or scheduling backups. You enable PITR on your table and your backup is available for restore at any point in time from the moment you enable it, up to a maximum of the 35 preceding days. For example, imagine a test script writing accidentally to a production DynamoDB table. You could recover your table to any point in time within the last 35 days.
+
+On-demand backups help with long-term archival requirements for regulatory compliance. On-demand backups give you full-control of managing the lifecycle of your backups, from creating as many backups as you need to retaining these for as long as you need.
+
+## Enabling PITR during deployment
+
+By default, the Cumulus [data-persistence module](https://github.com/nasa/cumulus/blob/master/tf-modules/data-persistence) enables PITR on the default tables listed in the [module's variable defaults](https://github.com/nasa/cumulus/blob/master/tf-modules/data-persistence/variables.tf) for `enable_point_in_time_tables`. At the time of writing, that list includes:
+
+- CollectionsTable
```

Comment:
Do we want to add Async Ops to this list?

Reply:
Yeah I can add it